pkg/apis/kudo/v1beta1/instance_types_helpers.go

```diff
@@ -145,35 +128,6 @@ func (i *Instance) PlanStatus(plan string) *PlanStatus {
 	return nil
 }
 
-// annotateSnapshot stores the current spec of Instance into the snapshot annotation
-// this information is used when executing update/upgrade plans, this overrides any snapshot that existed before
-func (i *Instance) AnnotateSnapshot() error {
```

Comment:
Fun fact: we don't need to save instance spec in the annotation anymore

Reply:
Fun fact 2: It's probably coming back for correct patching from the client side ;) But that doesn't concern the instance controller anymore, and it's good that it's gone here, having two versions in the annotations would be quite confusing

Reply:
Fun fact 3: I was hoping that we can avoid that and use `patchStrategy: replace` on the corresponding instance spec fields 😉  But yeah, having two of these would be very confusing.